id-repository/credential-service/src/main/java/io/mosip/credentialstore/util/IdrepositaryUtil.java

```diff
@@ -120,4 +124,75 @@ public IdResponseDTO getData(CredentialServiceRequestDto credentialServiceReques
 		}
 	}
 
+	public IdResponseDTO getDataById(CredentialServiceRequestDto credentialServiceRequestDto,

```

Comment:
What is the reason for duplicating the above method? @MuralitharanK

Reply:
@ase-101 https://mosip.atlassian.net/browse/ES-840